README.md

```diff
@@ -62,6 +62,32 @@ That's ok. `Quandl::Config` will find your config file if it's in a `config` fol
 
 If you want to provide the environment, use `ENV['RAILS_ENV']` or `ENV['RAKE_ENV']`. If you don't, the `default` environment will be assumed.
 
+### How do I override default values?
+
+You can override default values by adding a `configuration_options` method to your class/instance. The options available to be overwritten are:
+
+* `root_path` - Override this when you want to change the default path to your configuration files root folder. (default: your project path)
+* `environment` - Override this when you are not using a rails project and don't set the environment via `ENV['RAILS_ENV']`
+
+```ruby
+class A::B::Special
+  extend Quandl::Configurable
+  
+  def self.configuration_options
+    {
+        root_path: Pathname.new('~/configs/my_project'),
+        environment: ENV['MY_PROJECT']
+    }
+  end
+end
+```
+
+## FAQ
+
+### My config file won't reload in development mode (Rails)
+
+quandl-config utilizes an internal caching schema to save previously loaded configurations so that they don't load twice. You may need to restart your server in certain cases depending on how quandl-config was used in your project.
```

Comment:
Thats a bit trickier. The `clear_internal_cache` only works when your config has been included. If it has been extended then your definition is already locked to the class so the whole class would need to be redefined which is easiest by reloading the app.